src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/Target_X64/AssemblyJumpThunk.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The file naming convention seem to be "X64NodeType" so this would be X64SingleArgumentJumpThunk.cs". I don't necessarily agree with the convention, but let's be consistent.